<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color:silver
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color:aqua
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Bienvenido &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Admin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonAct">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color:white
}</string>
    </property>
    <property name="text">
     <string>actualizar producto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonElim">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color:white
}</string>
    </property>
    <property name="text">
     <string>eliminar producto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonVer">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color:white
}</string>
    </property>
    <property name="text">
     <string>ver productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>agregar producto</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
